### PR TITLE
docs(readme): use material.io URL

### PR DESCRIPTION
**Pull request by Splaktar**

### CODE REVIEW

**Comment by Splaktar** [comment]
@ebidel can you please merge this simple docs fix?

**Comment by ebidel** [comment]
@Splaktar I'm not working on this stuff anymore.

**Comment by Splaktar** [comment]
@justinfagnani can you or do you know anyone who can merge this simple docs PR?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #paper-elements
 
-The paper elements are a set of UI components designed to implement Google's [material design](http://www.google.com/design/spec/material-design/introduction.html) guidelines.
+The paper elements are a set of UI components designed to implement Google's [material design](https://material.io/guidelines/material-design/introduction.html) guidelines.
```
Comment (justinfagnani) [inline README.md]: Looks like this should be: https://material.io/design/introduction

**Comment by Splaktar** [comment]
@justinfagnani thank you for taking a look. URL updated.

**Review by justinfagnani — APPROVED** [review]

**Pull request merged**